lib/iamport.rb

```diff
@@ -48,92 +50,103 @@ def payments(options = {})
 
       uri = "payments/status/#{status}?page=#{page}"
 
-      _get(uri)
+      pay_get(uri)
     end
 
     # Find payment information using merchant uid
     # https://api.iamport.kr/#!/payments/getPaymentByMerchantUid
     def find(merchant_uid)
       uri = "payments/find/#{merchant_uid}"
 
-      _get(uri)
+      pay_get(uri)
     end
 
     # Prepare payment validation
     # https://api.iamport.kr/#!/payments/prepare
     def prepare(body)
       uri = "payments/prepare"
 
-      _post(uri, body)
+      pay_post(uri, body)
     end
 
     # Get prepared payment information by merchant_uid
     # https://api.iamport.kr/#!/payments/prepare/:merchant_uid
     def prepared(merchant_uid)
       uri = "payments/prepare/#{merchant_uid}"
 
-      _get(uri)
+      pay_get(uri)
     end
 
     # Canceled payments
     # https://api.iamport.kr/#!/payments/cancelPayment
     def cancel(body)
       uri = "payments/cancel"
 
-      _post(uri, body)
+      pay_post(uri, body)
     end
 
-    # Get a billing key by customer_uid
-    # GET https://api.iamport.kr/#!/subscribe/customers/:customer_uid
-    def find_subscribe_customer(customer_uid)
-      uri = "subscribe/customers/#{customer_uid}"
+    # create onetime
+    # DELETE https://api.iamport.kr/#!/subscribe/payments/onetime
+    def create_onetime_payment(payload = {})
```

Comment:
This comment seems not fit for the real action of the method. 
`DELETE` -> `POST`